server/store/datastore/server_config.go

```diff
@@ -0,0 +1,36 @@
+package datastore
+
+import "github.com/woodpecker-ci/woodpecker/server/model"
+
+func (s storage) ServerConfigGet(key string) (string, error) {
+	config := &model.ServerConfig{
+		Key: key,
+	}
+
+	err := wrapGet(s.engine.Get(config))
+	if err != nil {
+		return "", err
+	}
+
+	return config.Value, nil
```

Comment:
a simple get set test unit test would be nice

Reply:
done